starter-code/routes/drones.js

```diff
@@ -2,20 +2,36 @@ const express = require('express');
 
 // require the Drone model here
 
+const Drone = require('../models/drone')
 const router = express.Router();
 
-
-router.get('/drones', (req, res, next) => {
-  // Iteration #2
+router.get('/', (req, res, next) => {
+  console.log("Displaying drones");
+  Drone.find({}).then((drones) => {
+    res.render('drones/index', {
+      drones: drones,
+      title: 'Drone Database'
+    });
+  });
 });
 
 
-router.get('/drones/new', (req, res, next) => {
-  // Iteration #3
+router.get('/new', (req, res, next) => {
+  res.render('drones/new', {
+    drone: new Drone(),
```

Comment:
Os ha pasado a varios ya, no enviéis en el get un objeto new Drone() para el formulario, ya que debe estar vacío y recogeréis de ahí los datos.

Con renderizar el form os funciona perfectamente.